### PR TITLE
2019 spring event

### PR DESCRIPTION
**Pull request by ttdoda**

### CODE REVIEW

File: server/app/assets/javascript/map_image.coffee [hunk 1]
```diff
@@ -80,15 +117,77 @@ class MapImage
...
+        img.src = that.imageUrl + '?suffix=' + s
 
   setImage: () ->
     @ctx.globalAlpha = 1.0
-    @ctx.drawImage(@img, 0, 0, @img.width, @img.height)
+    that = @
+    if @bgLayers.length != 0
+      @bgLayers.forEach (name) ->
+        frame = that.frames[name]
+        that.ctx.drawImage(that.bgImg, frame.pos.x, frame.pos.y, frame.width, frame.height, 0, 0, frame.width, frame.height)
```
Comment (ponkotuy) [inline server/app/assets/javascript/map_image.coffee:171]: __[CoffeeLint]__ Line exceeds maximum allowed length Length is 124, max is 120 ([link](https://sider.review/gh/repos/16994149/pulls/434#issue-4040464))

<sub>posted by Sider</sub>

File: server/app/assets/javascript/route.coffee [hunk 2]
```diff
@@ -87,7 +93,7 @@ vueConf = (id) ->
       @dest = route.dest
       @setHash()
     modalURL: (route) ->
-      base = "/entire/sta/route_fleet/#{@area}/#{@info}/#{route.dep}/#{route.dest}"
+      base = if @is1st then jsRoutes.controllers.ViewSta.routeFleet1st(@area, @info, route.dep, route.dest).url else jsRoutes.controllers.ViewSta.routeFleet2nd(@area, @info, route.dep, route.dest).url
```
Comment (ponkotuy) [inline server/app/assets/javascript/route.coffee:96]: __[CoffeeLint]__ Line exceeds maximum allowed length Length is 200, max is 120 ([link](https://sider.review/gh/repos/16994149/pulls/434#issue-4040466))

<sub>posted by Sider</sub>

File: server/app/assets/javascript/map_image.coffee [hunk 3]
```diff
@@ -80,15 +117,77 @@ class MapImage
...
+    @layers.forEach (s) ->
+      that.setLayer(s)
+
+  setLayer: (s) ->
+    @ctx.globalAlpha = 1.0
+    that = @
+    Object.keys(@layerFrames[s]).forEach (name) ->
+      frame = that.layerFrames[s][name]
+      if frame.point
+        that.ctx.drawImage(that.layerImgs[s], frame.pos.x, frame.pos.y, frame.width, frame.height, frame.point.x, frame.point.y, frame.width, frame.height)
```
Comment (ponkotuy) [inline server/app/assets/javascript/map_image.coffee:186]: __[CoffeeLint]__ Line exceeds maximum allowed length Length is 155, max is 120 ([link](https://sider.review/gh/repos/16994149/pulls/434#issue-4040465))

<sub>posted by Sider</sub>

**Comment by ttdoda** [comment]
Pull-Requestの送り先を間違えた orz

**Pull request closed without merging**